src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-spring.profiles.include=@activatedProperties@
+spring.profiles.include=@activatedProperties@,${pgs.asyncMethod.profile:asyncMethodEnabled}
```

Comment:
Add `pgs.asyncMethod.profile` to map this from env variable and add the corresponding env variable to `.env.local`

Reply:
pgs.asyncMethod.profile is already an env varianble, and it is mapped to spring.profile.include property

I added pgs.asyncMethod.profile to env.local with value asyncMethodDisabled